exporters/BxDRobotExporter/BxDRobotExporter/BxDRobotExporter/StandardAddInServer.cs

```diff
@@ -334,8 +334,11 @@ private void EndExporter()
             // Export mesh as exporter is finished
             if (Utilities.GUI.SkeletonBase != null)
             {
-                Utilities.GUI.ExportMeshes();
-                Utilities.GUI.RobotSave();
+                if (Utilities.GUI.PromptSaveSettings(true, true))
```

Comment:
I don't mean to sound nitpicky but this `if` chain could be replaced by `&&`s. Functionally the same but might look a bit nicer.